packages/instrument-newrelic/package.json

```diff
@@ -38,7 +38,7 @@
     "access": "public"
   },
   "devDependencies": {
-    "@stellarjs/transport-redis": "^0.9.1",
+    "@stellarjs/queue-redis-bull": "^0.9.1",
```

Comment:
I'm not sure about the name, especially about `queue` (as it is the inner implementation and is implicitly conclude from `bull`)